.github/workflows/crowdin_upload.yml

```
@@ -3,7 +3,7 @@ name: Crowdin Upload
 on:
   push:
     paths: [ 'src/generated/resources/assets/*/lang/**' ]
-    branches: [ '1.20.4' ]
+    branches: [ '1.20.4', '1.20.5' ]
```

I think we only want to update crowdin from one branch source... **don't forget to update _crodwin_download.yml_**

```suggestion
    branches: [ '1.20.5' ]
```